tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -98,8 +98,57 @@ def _ensure_hash_in_reftest_screenshots(extra):
         if type(item) != dict:
             # Skip relation strings.
             continue
+        #TODO - Don't just take the final hash/screenshot, take a non-matching one
```

Comment:
Highlighting a TODO (I'm sure you're aware of it) that I think should be resolved before this is merged.